posts/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+# Create your models here.
+class Post(models.Model):
+    title = models.CharField(max_length=20)
+    content = models.TextField()
+    created = models.DateTimeField(auto_now=True)
+    modified = models.DateTimeField(auto_now_add=True)
+    author = models.ForeignKey('posts.Author', on_delete=models.CASCADE, null=True)
+
+    def __str__(self):
+        return self.title
+
+
+class Author(models.Model):
+    nick = models.CharField(max_length=15, unique=True)
+    bio = models.TextField(blank=True, null=True)
```

Comment:
Tworzenie pól tekstowych z blank=True i null=True to bad pattern, gdyż pozwala na 2 rózne sposoby zapisania "pustej" wartosci, do poczytania https://simpleisbetterthancomplex.com/tips/2016/07/25/django-tip-8-blank-or-null.html

Reply:
None, ""
Post.objects.exclude(bio__isnull=True, bio="")